### DISCUSSION TITLE
Pink and blue

### DISCUSSION THREAD

**Discussion by amoraleite** [keiwando/toonsquid#521] [Show and tell]
Hi guys! 

I would like to show my personal job made with ToonSquid. Take a look and tell me what you think! I'm still learning the great art of animation, but I'm proud of my progress!  

https://youtube.com/@PinkandBluecartoon?si=-SLdnstBOMXvhpO-

Thanks:)

**Comment by andydjames** [comment]
Nice. How did you do the blur effect in "Quer dicas para a foto de casal sair boa?". In After Effects?

**Reply by amoraleite** [reply]
Hi! No, I’m using the CapCut. But jus for the final edition and finishing. This blur is a vídeo effect! 😃
